ocaml/sdk-gen/README.md

```diff
@@ -16,8 +16,9 @@ README files. The (patched) third party libraries required for the compilation
 of the C# and PowerShell source code can be obtained from
 [dotnet-packages][https://github.com/xenserver/dotnet-packages].
```

Comment:
Done. I believe the `xenserver-samples` don't have a link to this repo - but should not need it anyway, because the instruction will be to use the dlls from the released library.